deployment/docker-compose/README.md

```diff
@@ -0,0 +1,458 @@
+
+### Standalone Installation
+This section walk through the instructions to install Alert in a standalone fashion.
+#### Overview
+1. Create a directory for secrets.
+2. Create ALERT_ENCRYPTION_PASSWORD file.
+3. Create ALERT_ENCRYPTION_GLOBAL_SALT file.
+4. Manage certificates.
+5. Modify environment variables.
+6. Bring the containers up.
+ 
+#### Details 
+This section walks through each step of the installation procedure.
+
+##### 1. Create a directory for secrets.
+
+- Create a directory to store secrets.
+    ```
+    mkdir -p <PATH>
+    ```
+    
+    Example: 
+    ```
+    mkdir -p /alert/mysecrets
+    ```
+    
+- Uncomment the following from the docker-compose.local-overrides.yml file alert service section.
+    ```
+      volumes: ['<PATH_TO_SECRETS>:/run/secrets']
+    ```
+      
+- Replace <PATH_TO_SECRETS> of the docker-compose.local-overrides.yml file with the directory just created.
+    
+    Example:
+    ```
+        volumes:['/alert/mysecrets:/run/secrets']
+    ```
+##### 2. Create ALERT_ENCRYPTION_PASSWORD file.
+  
+- Create a file containing the encryption password for Alert in the secrets directory.
+
+    ```
+    echo "<PASSSWORD_TEXT>" >> <PATH_TO_SECRETS>/ALERT_ENCRYPTION_PASSWORD
+    ```
+
+##### 3. Create ALERT_ENCRYPTION_GLOBAL_SALT file.
+
+- Create a file containing the encryption salt for Alert in the secrets directory.
+
+    ```
+    echo "<SALT_TEXT>" >> <PATH_TO_SECRETS>/ALERT_ENCRYPTION_GLOBAL_SALT
+    ```
+
+##### 4. Manage certificates.
+This is an optional step. Confirm if custom certificates or a certificate store need to be used.
+- Using custom certificate for Alert web server. See [Using Custom Certificates](#using-custom-certificates)
+- Using custom trust store to trust certificates of external servers. See [Using Custom Certificate TrustStore](#using-custom-certificate-truststore)
+
+#### 5. Modify environment variables.
+Please see [Environment Variables](#environment-variables)
+- Set the required environment variable ALERT_HOSTNAME. See [Alert Hostname Variable](#alert-hostname-variable)
+- Set any other optional environment variables as needed.
+
+##### 6. Bring the containers up.
+- Start the containers 
+    ```
+    docker-compose -f ./docker-compose/docker-compose.yml -f ./docker-compose/docker-compose.local-overrides.yml -p <PROFILE_NAME> up -d
+    ```
+- Be sure to replace <PROFILE_NAME> with the profile name you wish to use for your deployment i.e. blackduck.
+    ```
+    docker-compose -f ./docker-compose/docker-compose.yml -f ./docker-compose.local-overrides.yml -p blackduck up -d
+    ```
+    
+    Note: Don't forget the -d option at the end of the command line to run the command as a daemon process otherwise the container logs will go to standard output and ```ctrl+c``` will stop the application.
+    
+### Installation with Black Duck
+Overview:
+1. Create a directory for secrets.
+2. Create ALERT_ENCRYPTION_PASSWORD file.
```

Comment:
Is this the name of the file?

Reply:
Yes BlackDuck also uses capital letters in its file names for secrets.  For Alert we modeled the same mechanism and the file names match the environment variables in order to avoid confusion and be consistent.